Assets/Scripts/UI/Menu/MainMenu.cs

```diff
@@ -1,14 +1,7 @@
-﻿using UnityEngine; 
-
-namespace Assets.Scripts.UI.Menu
+﻿namespace Assets.Scripts.UI.Menu
 {
     public class MainMenu : UiNavigationElement
     {
-        private void OnEnable()
-        {
-            Cursor.visible = true;
-            Cursor.lockState = CursorLockMode.Confined; 
```

Comment:
Reminds me that we should try using `Confined` in Original and WOW. It caused a bug earlier, maybe not anymore?

Reply:
So investigate and confirm if there's still a bug or not

Reply:
It still caused a bug, but I just fixed that.